app/src/main/java/com/example/memeitupapp/ui/mainmemes/MainMemesActivity.kt

```diff
@@ -0,0 +1,33 @@
+package com.example.memeitupapp.ui.mainmemes
+
+import android.os.Bundle
+import androidx.appcompat.app.AppCompatActivity
+import androidx.viewpager2.widget.ViewPager2
+import com.example.memeitupapp.R
+import com.example.memeitupapp.ui.adapter.ViewPagerFragmentAdapter
+import com.google.android.material.tabs.TabLayout
+import com.google.android.material.tabs.TabLayoutMediator
+import kotlinx.android.synthetic.main.activity_main.pager
+import kotlinx.android.synthetic.main.activity_main.tabLayout
+
+class MainMemesActivity : AppCompatActivity() {
+
+    override fun onCreate(savedInstanceState: Bundle?) {
+        super.onCreate(savedInstanceState)
+        setContentView(R.layout.activity_main)
+
+        initViewPager2WithFragments()
+    }
+
+    private fun initViewPager2WithFragments() {
```

Comment:
it is necessary the name ViewPager ""2"" ?

Reply:
no it's no, removed, forgot to change it, thanks